### DISCUSSION TITLE
Install for all users on one PC

### DISCUSSION THREAD

**Discussion by joe-oli** [stashapp/stash#6880] [General]
### Scope
I installed Stash while I was logged in remotely via RDP (say, user-remote); When I logged in locally, under another user (say, user-local), I end up on the http://localhost:9999/setup page again, where I have to reconfigure again, scan again, etc.

##  Long Form
Naturally, this is to be expected, as the default user config yaml files are in the C:\Users\<specific-user>;
So I copied the C:\Users\user-remote\\.stash folder onto C:\Users\user-local\\.stash, and made some edits to the config.yml file, but I am still being asked to reconfig, scan again.

## Wiki Topic
What would be good is a Wiki topic covering this issue, or a `How To` instructions on what to do exactly, such that the use-case I described is catered for, i.e. install is for the machine, not a specific user.

**Comment by philpw99** [comment]
Well, maybe put the files in C:\Stash and copy the config yaml files to that folder. This way you should be able to get the same result for all the users.

**Comment by WithoutPants** [comment]
It's documented in the setup page, but I'm not sure it is elsewhere: when you start stash, it first looks for `config.yml` in the current working directory. If it doesn't find it there, it looks in `$HOME/.stash`. If it can't find it there, then it assumes that it's a new system and runs the setup wizard. The wizard asks you where to put the configuration as the first step.

The solution to your issue is likely to move the contents to a folder that is accessible to all users, _and copy_ the stash exe file to that same directory. When you run stash from that directory, it should pick up the `config.yml` file from that directory.
